tests/network/connectivity/test_pod_network.py

```diff
@@ -73,12 +74,30 @@ def cloud_init_ipv6_network_data(dual_stack_network_data):
     return compose_cloud_init_data_dict(ipv6_network_data=dual_stack_network_data)
 
 
+@pytest.mark.parametrize(
+    "ip_family",
+    [
+        pytest.param(
+            IPV4_STR,
+            marks=[
+                pytest.mark.polarion("CNV-2332"),
+                pytest.mark.ipv4,
+            ],
+        ),
+        pytest.param(
```

Comment:
Maybe I'm missing something, but if you implement both IP families in the same test using parameterize, in the IPv6 scenario you won't get the test failing if the cluster doesn't support IPv6, no? They should be separated to different tests so that the IPv6 scenario can call fail_if_not_ipv6_supported_cluster.

Reply:
Look at the parameterization, the test was splitted to 2 tests - CNV-2332 for IPv4 and CNV-11845 for IPv6.
Please resolve the comment so it won't block the merge.